adserver/tasks.py

```diff
@@ -668,6 +670,27 @@ def calculate_publisher_ctrs(days=7):
         publisher.save()
 
 
+@app.task()
+def calculate_ad_ctrs(days=7):
+    """Calculate sampled CTRs for all active ads for the last X days."""
+    sample_cutoff = get_ad_day() - datetime.timedelta(days=days)
+
+    for ad in Advertisement.objects.filter(live=True, flight__live=True):
```

Comment:
I wonder if we just want to filter by a baseline of views (or clicks?) in the past X days? Seems like we could get some weird outcomes here if we don't have a baseline of at least 1000 views in the period. 

I guess the weighting doesn't throw things too heavily off, but if I have 1 view and 0 clicks (or 5 views and 1 click), my CTR is not going to be representative.

Reply:
That seems reasonable to require at least 1000 views in the period.